lib/compiler/src/target.rs

```diff
@@ -160,9 +161,11 @@ impl ToString for CpuFeature {
 
 /// This is the target that we will use for compiling
 /// the WebAssembly ModuleInfo, and then run it.
-#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, MemoryUsage)]
 pub struct Target {
+    #[memoryusage(ignore)]
     triple: Triple,
+    #[memoryusage(ignore)]
```

Comment:
Why ignoring this?

Reply:
Because we don't own this type. We can't implement `MemoryUsage` on this type for the moment. We may want to implement `MemoryUsage` manually on `Target` and try a custom computation though.